### ISSUE TITLE
Error: Cannot find module './format'

### ISSUE THREAD

**Issue by bravehack** [personality-insights/text-summary#3]
Get that error after including the package as a nodejs module. Installing the 'format' node module doesn't resolve the issue.

**Comment by germanattanasio** [comment]
I pushed a fix to github, can you update your package.json and point to this repo instead of specifying a version?
```
...
personality-text-summary: 'https://github.com/personality-insights/text-summary'
...
```

If it works I will publish the release

**Comment by bravehack** [comment]
Thank you soo much! It's better.
 
I'm now getting the same error for the next line :
 
Error: Cannot find module './i18n/en'
 
(I was afraid I would, but couldn't passed the ./format part.)
 
Best regards,
Rob

**Comment by bravehack** [comment]
German,
 
Thank you. I got it to work by providing ./i18n/en.js and ./i18n/es.js and commenting out the line that requires ja.js in the i18n.js file.
 
I still did not get it to work with V3 of Personality Insights.
 
So your fix addressed the primary problem that would have kept me stumped for days.  THANK YOU!!!
 
Have a great evening!

**Comment by germanattanasio** [comment]
This was fixed in v2.0.3. @bravehack feel free to open the issue again if this is not the case.

The new version also supports the new v3 API and should be backward compatible
